docs/en/tools/visualization.md

```diff
@@ -2,29 +2,29 @@
 
 <!-- TOC -->
 
-- [Visualization](#visualization)
-  - [Pipeline Visualization](#pipeline-visualization)
-  - [Learning Rate Schedule Visualization](#learning-rate-schedule-visualization)
-  - [Class Activation Map Visualization](#class-activation-map-visualization)
-  - [FAQs](#faqs)
+- [Pipeline Visualization](#pipeline-visualization)
+- [Learning Rate Schedule Visualization](#learning-rate-schedule-visualization)
+- [Class Activation Map Visualization](#class-activation-map-visualization)
+- [FAQs](#faqs)
 
 <!-- TOC -->
 ## Pipeline Visualization
 
 ```bash
 python tools/visualizations/vis_pipeline.py \
     ${CONFIG_FILE} \
-    --output-dir ${OUTPUT_DIR} \
-    --phase ${DATASET_PHASE} \
-    --number ${BUNBER_IMAGES_DISPLAY} \
-    --skip-type ${SKIP_TRANSFORM_TYPE}
-    --mode ${DISPLAY_MODE} \
-    --show \
-    --adaptive \
-    --min-edge-length ${MIN_EDGE_LENGTH} \
-    --max-edge-length ${MAX_EDGE_LENGTH} \
-    --bgr2rgb \
-    --window-size ${WINDOW_SIZE}
+    [--output-dir ${OUTPUT_DIR}] \
+    [--phase ${DATASET_PHASE}] \
+    [--number ${BUNBER_IMAGES_DISPLAY}] \
+    [--skip-type ${SKIP_TRANSFORM_TYPE}] \
+    [--mode ${DISPLAY_MODE}] \
+    [--show] \
+    [--adaptive] \
+    [--min-edge-length ${MIN_EDGE_LENGTH}] \
+    [--max-edge-length ${MAX_EDGE_LENGTH}] \
+    [--bgr2rgb] \
+    [--window-size ${WINDOW_SIZE}] \
+    [--cfg-options ${CFG_OPTIONS}]
 ```
 
 **Description of all arguments**：
```

Comment:
#35 to be visualized
#39 If set, adaptively resize images for better visualization.

Reply:
done.